app/mailers/user_mailer.rb

```diff
@@ -9,4 +9,12 @@ def welcome(options = {})
       mail(:to => user.mail, :subject => _("Welcome to Foreman"))
     end
   end
+
+  def tester(options = {})
+    user = options[:user]
+
+    set_locale_for(user) do
+      mail(:to => user.mail, :subject => _("Foreman Mailer Test"))
```

Comment:
Yeah I noticed, fixing in my comment. Do we want to follow that pattern in the email itself? Changing that as well.